.eslintrc.yml

```diff
@@ -11,3 +11,4 @@ rules:
   "comma-dangle": ["error", "never"]
   func-names: ["error", "never"]
   linebreak-style: ["error", "unix"]
+  no-unused-expressions: 0
```

Comment:
I disagree with this change, put in 0 this property removes several rules. https://eslint.org/docs/rules/no-unused-expressions

Also, you only disable that to Chai asserts then disable directly into the test (like before Yesid comment). or search a library in npm with Chai eslint rules
@leonleo997